src/iop/denoiseprofile.c

```diff
@@ -2987,9 +3071,9 @@ void commit_params(dt_iop_module_t *self,
   if(p->a[0] == -1.0)
   {
     // autodetect matching profile again, the same way as detecting their names,
-    // this is partially duplicated code and data because we are not allowed to access
-    // gui_data here ..
-    dt_noiseprofile_t interpolated = dt_iop_denoiseprofile_get_auto_profile(self);
+    // because we are not allowed to access gui_data here ..
+    dt_noiseprofile_t interpolated
+        = dt_iop_denoiseprofile_get_auto_profile(self, NULL, NULL, 0, NULL, p->compensate_hilite_pres);
```

Comment:
style: `=` should be above

Reply:
clang-format reverts the change if I move the `=` to the line above, am I holding it wrong?

Reply:
I see. Shall I make a PR to update [CONTRIBUTING.md#coding-style](https://github.com/darktable-org/darktable/blob/master/CONTRIBUTING.md#coding-style) then? (that was why I had it enabled in the first place, I don't usually use it)

Reply:
> Shall I make a PR to update [CONTRIBUTING.md#coding-style](https://github.com/darktable-org/darktable/blob/master/CONTRIBUTING.md#coding-style) then? (that was why I had it enabled in the first place, I don't usually use it)

The file in question does not prompt or require you to enable clang-format. It says the following:
`You can enforce your commits to follow it` (note: "you can", not "you must")
So yes, you can do as per the instructions in that file. If you want. There is no written requirement to run clang-format.

Rather, we should check the contents of .clang-format and add options that more fully reflect the current code style.